orion-ui/.eslintrc.js

```diff
@@ -4,9 +4,12 @@ module.exports = {
     node: true
   },
   extends: ["@prefecthq"],
+  parser: 'vue-eslint-parser',
   parserOptions: {
     ecmaVersion: 2020,
-    project: "./tsconfig.json"
+    project: ["./tsconfig.json"],
+    sourceType: 'module',
+    parser: '@typescript-eslint/parser'
```

Comment:
Can you give a brief explanation of these changes? Just wanting to understand why these were needed when they weren't before.

Reply:
the important thing here is we're explicitly using the `vue-eslint-parser`, which has a parser option of the underlying parser. 

without this change, running `npm run lint` would error because the rules we specify required "project" set in parser options. With some time on github issues, I discovered this solution.

Reply:
Nice, thanks!